package.json

```diff
@@ -24,6 +24,7 @@
     "es6-promisify": "^6.0.0",
     "grommet": "^1.1.0",
     "prop-types": "^15.5.8",
+    "query-string": "^6.1.0",
```

Comment:
Can you also add @types/query-string please ?

Reply:
query-string is causing build problems so it is not used anymore. Cf the next commit 9259d01